tests/ecosystem/deployment/test_deployment.py

```diff
@@ -78,3 +82,25 @@ def test_deployment(pvc_factory, pod_factory):
 
     if teardown:
         log.info("Cluster will be destroyed during teardown part of this test.")
+
+
+@azure_platform_required
+def test_azure_storageaccount():
+    """
+    Testing that Azure storage account, post deployment.
+
+    Testing for property 'allowBlobPublicAccess' to be 'false'
+    """
+    azure = azure_utils.AZURE()
+    storage_account_names = azure.get_storage_accounts_names()
+    print(f"{storage_account_names=}")
+    for storage in storage_account_names:
+        if "noobaaaccount" in storage:
+            property = str(azure.get_storage_account_properties(storage))
+            pat = r"'allow_blob_public_access': (True|False),"
+
+            from re import findall
+
+            match = findall(pat, property)
+
+            assert match[0] == "False"  # checking if the matched value is False or not
```

Comment:
If re.findall() finds 0 match strings, it will return an empty list [].
in such case assert match[0] == "False" will fail with IndexError. Also will be good to print failure message after the condition.

Reply:
Added those!